### ISSUE TITLE
[1.4.0] fails to build on hurd-i386

### ISSUE THREAD

**Issue by cdluminate** [hpjansson/chafa#40]
```shell
/bin/bash ../../libtool  --tag=CC   --mode=link gcc -Wall -Wextra -Wmissing-prototypes -ffast-math -fvisibility=hidden -I/usr/include/glib-2.0 -I/usr/lib/i386-gnu/glib-2.0/include -fopenmp -DMAGICKCORE_HDRI_ENABLE=0 -DMAGICKCORE_QUANTUM_DEPTH=16 -fopenmp -DMAGICKCORE_HDRI_ENABLE=0 -DMAGICKCORE_QUANTUM_DEPTH=16 -I/usr/include/i386-gnu//ImageMagick-6 -I/usr/include/ImageMagick-6 -I/usr/include/i386-gnu//ImageMagick-6 -I/usr/include/ImageMagick-6 -I/usr/include/freetype2 -I/usr/include/libpng16 -g -O2 -fdebug-prefix-map=/<<PKGBUILDDIR>>=. -fstack-protector-strong -Wformat -Werror=format-security  -Wl,-z,relro -Wl,-z,now -Wl,--as-needed -o chafa chafa-chafa.o chafa-file-mapping.o chafa-font-loader.o chafa-gif-loader.o chafa-named-colors.o chafa-xwd-loader.o -lglib-2.0 -lMagickWand-6.Q16 -lMagickCore-6.Q16 -lfreetype ../../chafa/libchafa.la ../../libnsgif/libnsgif.la 
libtool: link: gcc -Wall -Wextra -Wmissing-prototypes -ffast-math -fvisibility=hidden -I/usr/include/glib-2.0 -I/usr/lib/i386-gnu/glib-2.0/include -fopenmp -DMAGICKCORE_HDRI_ENABLE=0 -DMAGICKCORE_QUANTUM_DEPTH=16 -fopenmp -DMAGICKCORE_HDRI_ENABLE=0 -DMAGICKCORE_QUANTUM_DEPTH=16 -I/usr/include/i386-gnu//ImageMagick-6 -I/usr/include/ImageMagick-6 -I/usr/include/i386-gnu//ImageMagick-6 -I/usr/include/ImageMagick-6 -I/usr/include/freetype2 -I/usr/include/libpng16 -g -O2 -fdebug-prefix-map=/<<PKGBUILDDIR>>=. -fstack-protector-strong -Wformat -Werror=format-security -Wl,-z -Wl,relro -Wl,-z -Wl,now -Wl,--as-needed -o .libs/chafa chafa-chafa.o chafa-file-mapping.o chafa-font-loader.o chafa-gif-loader.o chafa-named-colors.o chafa-xwd-loader.o  -lglib-2.0 /usr/lib/i386-gnu/libMagickWand-6.Q16.so /usr/lib/i386-gnu/libMagickCore-6.Q16.so -lfreetype ../../chafa/.libs/libchafa.so ../../libnsgif/.libs/libnsgif.a -pthread -fopenmp
/usr/bin/ld: ../../chafa/.libs/libchafa.so: undefined reference to `_mm_cvtm64_si64'
collect2: error: ld returned 1 exit status
Makefile:451: recipe for target 'chafa' failed
```
https://buildd.debian.org/status/fetch.php?pkg=chafa&arch=hurd-i386&ver=1.4.0-1&stamp=1586408244&raw=0
https://buildd.debian.org/status/package.php?p=chafa

**Comment by cdluminate** [comment]
The same happens on i386
https://buildd.debian.org/status/fetch.php?pkg=chafa&arch=i386&ver=1.4.0-1&stamp=1586411853&raw=0

**Comment by hpjansson** [comment]
Addressed by commit 9c18cb9806d8de3952fb5dc6b285fbff7069f48e. Also cherry-picked to 1.4 branch. I can make a 1.4.1 release later today or tomorrow (also with the rpath fix) so you can skip directly to packaging that.

**Comment by hpjansson** [comment]
I just released 1.4.1 from the 1.4 branch. Hopefully that's everything!

**Comment by cdluminate** [comment]
successfully built on all supported architectures (excluding kfreebsd which lacks build dependencies)
